python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -36,6 +36,7 @@ class BulkActionStatus(Enum):
     CANCELED = "CANCELED"
     COMPLETED_SUCCESS = "COMPLETED_SUCCESS"
     COMPLETED_FAILED = "COMPLETED_FAILED"  # denotes that the backfill daemon completed successfully, but some runs failed
+    DELETING = "DELETING"
```

Comment:
todo: need to figure out how this status will be displayed in the UI since we convert backfill statuses to corresponding run statuses but there is no run status that aligns perfectly